### PR TITLE
Add badge print table at edit ticket page

### PR DESCRIPTION
**Pull request by tomrndom**
image
image

ref: https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskProject&view=vertical&task=3389453&stafilter=NotComplete&fileview=grid

Signed-off-by: Tomás Castillo <tcastilloboireau@gmail.com>

### CODE REVIEW

File: src/reducers/tickets/badge-print-reducer.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+const badgePrintReducer = (state = DEFAULT_STATE, action) => {
+    const { type, payload } = action
+    switch (type) {
+        case LOGOUT_USER: {
+            // we need this in case the token expired while editing the form
+            if (payload.hasOwnProperty('persistStore')) {
+                return state;
+            } else {
+                return { ...state, entity: { ...DEFAULT_ENTITY } };
```
Comment (smarcet) [inline src/reducers/tickets/badge-print-reducer.js:40]: DEFAULT_ENTITY is not defined ?
what is the purpose of catching the LOGUT_USER action /

**Review by smarcet — CHANGES_REQUESTED** [review]
@tomrndom review comments

File: src/reducers/tickets/badge-print-reducer.js (hunk 1 above)
Comment (tomrndom) [reply src/reducers/tickets/badge-print-reducer.js]: Fixed issue with variable not defied.
LOGOUT_USER catch is to keep consistency with all the reducers at logout

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**